config/environments/development.rb

```diff
@@ -54,23 +62,25 @@
   config.assets.quiet = true
 
   # Raises error for missing translations.
-  config.action_view.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = true
```

Comment:
Did you mean to comment out this line?

Reply:
Great catch! This was unintentional 😅. I'll update this.

Reply:
Done!